src/DetailsView/components/load-assessment-button.tsx

```diff
@@ -3,14 +3,46 @@
 
 import * as React from 'react';
 import { InsightsCommandButton } from 'common/components/controls/insights-command-button';
-export interface LoadAssessmentButtonProps {}
+import { DetailsViewActionMessageCreator } from 'DetailsView/actions/details-view-action-message-creator';
+import { AssessmentDataParser } from 'common/assessment-data-parser';
+
+export type LoadAssessmentButtonDeps = {
+    detailsViewActionMessageCreator: DetailsViewActionMessageCreator;
+    assessmentDataParser: AssessmentDataParser;
+};
+export interface LoadAssessmentButtonProps {
+    deps: LoadAssessmentButtonDeps;
+}
 
 export class LoadAssessmentButton extends React.Component<LoadAssessmentButtonProps> {
     public render(): JSX.Element {
         return (
-            <InsightsCommandButton iconProps={{ iconName: 'FabricOpenFolderHorizontal' }}>
+            <InsightsCommandButton
+                iconProps={{ iconName: 'FabricOpenFolderHorizontal' }}
+                onClick={this.getAssessmentForUpload}
+            >
                 Load assessment
             </InsightsCommandButton>
         );
     }
+
+    private getAssessmentForUpload = () => {
```

Comment:
Given that this button says "Load assessment", I'm wondering if "upload" is the right word to use in the code here and in the other places in this PR (`UploadAssessmentPayload`, `onUploadAssessment`, etc). Do you have a preference for upload?

Reply:
You're right, the naming here is inconsistent, will update `upload` to `load`. Thanks for catching this!